Router/Router.go

```diff
@@ -0,0 +1,33 @@
+package Router
+
+import (
+	"business/Controllers/CustomersController"
+	"business/Controllers/OrdersController"
+	"business/Controllers/ProductsController"
```

Comment:
try to implement authentication

Reply:
Can u provide some example code that shows the best practice for performing authentication